common/views/components/Promo/Promo.js

```diff
@@ -165,7 +170,14 @@ const Promo = ({
           }
 
           {description &&
-            <span className="promo__copy">{truncate(striptags(description), 140)}</span>
+            <span className={`inline-block ${font({s: 'HNL4'})} ${spacing({s: 1}, {margin: ['bottom']})}`}>{truncate(striptags(description), 140)}</span>
+          }
+
+          {(commissionedSeries || seriesTitle) &&
+            <span className={`block font-charcoal ${spacing({s: 1}, {margin: ['bottom']})} ${font({s: 'HNL6', l: 'HNL5'})}`}
+              aria-hidden='true'>
+              {partOf(commissionedSeries, seriesTitle)}
+            </span>
           }
         </div>
       </PromoDescription>
```

Comment:
We need to break this component up.

Reply:
Agree. I think we need to decoupled the 'standard' and work promos, first.